cmd/ttn-lw-cli/internal/util/flags.go

```diff
@@ -684,6 +684,14 @@ func setField(rv reflect.Value, path []string, v reflect.Value) error {
 								},
 							}))
 						}
+					case ft.Elem().PkgPath() == "go.thethings.network/lorawan-stack/v3/pkg/types" &&
+						ft.Elem().Name() == "DevAddrPrefix" && vt.Elem().Kind() == reflect.String:
+						for i := 0; i < v.Len(); i++ {
+							err := slice.Index(i).Addr().Interface().(encoding.TextUnmarshaler).UnmarshalText([]byte(v.Index(i).String()))
+							if err != nil {
+								return err
+							}
+						}
 					case ft.Elem().Kind() == reflect.Int32 && vt.Elem().Kind() == reflect.String:
```

Comment:
Can't we avoid the block above if we simply change this?

```suggestion
					case vt.Elem().Kind() == reflect.String:
```

Reply:
Yup. That works.